manifests/t/toasterofbread/spmp/0.4.1/toasterofbread.spmp.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: toasterofbread.spmp
+PackageVersion: 0.4.1
+InstallerType: msi

```

```suggestion
InstallerType: exe
```
Hi @KaranKad,

The package manager changes the download extension to match the `InstallerType`, for some. This might require `InstallerSwitches` to be defined.